### PR TITLE
Use validation set for evaluation

### PR DESCRIPTION
**Pull request by fatosmorina**
I noticed that at the very end of the example, you were using the training set for both training and validation, probably because of a typo.

### CODE REVIEW

**Comment by QinbinLi** [comment]
Hi @fatosmorina ,

Thanks! The file is an example provided by LightGBM ([here](https://github.com/microsoft/LightGBM/blob/master/examples/python-guide/advanced_example.py)) and we didn't modify it. In their code, they use the training dataset for validation in the first 10 rounds. This file is not used in the training of DPBoost.
